package.json

```diff
@@ -64,7 +64,7 @@
     "babel-eslint": "10.1.0",
     "eslint": "7.32.0",
     "eslint-config-kentcdodds": "19.1.1",
-    "eslint-config-next": "11.0.1",
+    "eslint-config-next": "11.1.0",
```

Comment:
> ## ESLint Improvements
> 
> In Next.js 11, we introduced built-in ESLint support through `next lint`. Since its initial release, we've continued adding rules that help developers fix common mistakes in their applications.

Is that command something we could use/benefit from somehow?

https://nextjs.org/blog/next-11-1#eslint-improvements

---

Also this seems nice:

> ## Default Accessibility Rules
> 
> Better accessibility rules are now included by default, preventing issues with ARIA properties that do not match each other and usage of non-existent ARIA attributes. These rules will warn by default.

---

We also get nice image optimization improvements. I notice that loading the big images always takes a while, maybe we can use that `sharp` package, I'd trade a slower build for faster image generation on the production side... But maybe I'm misunderstanding something.

> ## Image Optimization
>
> By default, Next.js uses WebAssembly to perform Image Optimization, which offsets the install time of the Next.js package by being significantly smaller and not having a post-install step. With Next.js 11.1, you can optionally install `sharp` which optimizes for uncached image generation time with a trade-off of slower installation.

https://nextjs.org/blog/next-11-1#image-optimization

Reply:
-  ESLint plugin for Next was already set up when updating to v11: https://github.com/Belco90/octoclairvoyant/pull/190/files#diff-e2954b558f2aa82baff0e30964490d12942e0e251c1aa56c3294de6ec67b7cf5R16 It can run ESLint through `next link` but since we already have our own ESLint setup, it's better just to load their plugin to extend their rules.
- a11y rules should be enabled now since we are already using their ESLint plugin
- We can check the Image Optimization with `sharp` but better to do so in a separate PR.

Reply:
ESLint and a11y are good to go.

We can try out the `sharp` image optimization in a new PR.